### PR TITLE
LPS-148457 Sign In page do not display correctly.

### PR DESCRIPTION
**Pull request by david-gutierrez-mesa**
https://issues.liferay.com/browse/LPS-148457

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro [hunk 1]
```diff
@@ -49,6 +49,10 @@ definition {
 		Click(locator1 = "Button#SUBMIT");
 	}
 
+	macro goBack {
+		RunScript(locator1 = "window.history.go(-1)");
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro:53]: do we have a task to track do it well once they implement https://issues.liferay.com/browse/POSHI-302

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase [hunk 2]
```diff
@@ -1110,6 +1110,62 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-148457. Masterclass create account and forgot password are not displayed in sign in page"
+	@priority = "3"
+	test MasterclassSignInPageDoesNotDisplayCreateAndForgotPage {
+		property portal.acceptance = "true";
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:1116]: do we want to add acceptance to a propriety 3 test?

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro [hunk 3]
```diff
@@ -49,6 +49,10 @@ definition {
 		Click(locator1 = "Button#SUBMIT");
 	}
 
+	macro goBack {
```
Comment (david-gutierrez-mesa) [inline modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro:52]: should we specify in the name that is using history or back button?

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro (hunk 1 above)
Comment (AlfredSoto) [reply modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro:53]: I've linked POSHI task with this Testing Story. But I'll create a task to not forget to update the test after poshi task is done.

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase (hunk 2 above)
Comment (AlfredSoto) [reply modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/tests/Masterclass.testcase:1116]: Nop. copy&paste error.

File: modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro (hunk 3 above)
Comment (AlfredSoto) [reply modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro:52]: Yes, We can do it. In fact, we can have both methods. goBackUsingHistory and goBackUsingBrowserArrow (when poshi task is done)

Comment (david-gutierrez-mesa) [reply modules/apps/site-initializer/site-initializer-masterclass/site-initializer-masterclass-test/src/testFunctional/macros/Masterclass.macro:52]: make sense!

**Comment by AlfredSoto** [comment]
Closed. See https://github.com/david-gutierrez-mesa/liferay-portal/pull/302

**Comment by AlfredSoto** [comment]
ci:close

**Pull request closed without merging**